### PR TITLE
[KOGITO-7839] Align the apiVersion to the Kogito API Implementation o…

### PR DESCRIPTION
**Pull request by davidesalerno**
…f the SW Spec

Signed-off-by: Davide Salerno <dsalerno@redhat.com>

**Description of the change:**

**Motivation for the change:**

**Checklist**

- [x] Add or Modify a unit test for your change
- [x] Have you verified that tall the tests are passing?

### CODE REVIEW

File: bundle/manifests/kogito-serverless-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-apiVersion: operators.coreos.com/v1alpha1
+apiVersion: operators.coreos.com/v08
```
Comment (ricardozanini) [inline bundle/manifests/kogito-serverless-operator.clusterserviceversion.yaml:1]: I think we can't change this.

**Review by ricardozanini — COMMENTED** [review]
Also, the namespace should be `sw.kogito.kie.org`, correct?

File: bundle/manifests/kogito-serverless-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (davidesalerno) [reply bundle/manifests/kogito-serverless-operator.clusterserviceversion.yaml:1]: +1

File: controllers/kogitoserverlessworkflow_controller.go [hunk 2]
```diff
@@ -18,7 +18,7 @@ package controllers
 
 import (
 	"context"
-	serverlessv1alpha1 "github.com/davidesalerno/kogito-serverless-operator/api/v1alpha1"
+	serverlessv08 "github.com/davidesalerno/kogito-serverless-operator/api/v08"
```
Comment (spolti) [inline controllers/kogitoserverlessworkflow_controller.go:21]: https://github.com/kiegroup/kogito-operator/pull/1281

File: main.go [hunk 3]
```diff
@@ -31,7 +31,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/healthz"
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 
-	serverlessv1alpha1 "github.com/davidesalerno/kogito-serverless-operator/api/v1alpha1"
+	serverlessv08 "github.com/davidesalerno/kogito-serverless-operator/api/v08"
```
Comment (spolti) [inline main.go:34]: same.
or we can do the opposite, use serverlessv08 on other imports as well.
btw, is there a nickname for serverless, maybe srvlss?

**Review by spolti — APPROVED** [review]

File: controllers/kogitoserverlessworkflow_controller.go (hunk 2 above)
Comment (davidesalerno) [reply controllers/kogitoserverlessworkflow_controller.go:21]: Are you suggesting to switch from serverlessv08 to something like apiv08?

Comment (spolti) [reply controllers/kogitoserverlessworkflow_controller.go:21]: lol sry, I don't know how this link ended here
see my comment below.

**Review by desmax74 — APPROVED** [review]

**Review by spolti — APPROVED** [review]

File: config/manifests/bases/kogito-serverless-operator.clusterserviceversion.yaml [hunk 4]
```diff
@@ -15,7 +15,7 @@ spec:
       displayName: Kogito Serverless Workflow
       kind: KogitoServerlessWorkflow
       name: kogitoserverlessworkflows.kie.kogito.sw.org
```
Comment (ricardozanini) [inline config/manifests/bases/kogito-serverless-operator.clusterserviceversion.yaml:17]:
```suggestion
      name: kogitoserverlessworkflows.sw.kogito.kie.org
```

I think we need to review the naming.

**Pull request merged**